src/structures/BaseGuildVoiceChannel.js

```diff
@@ -66,30 +66,6 @@ class BaseGuildVoiceChannel extends GuildChannel {
     return true;
   }
 
-  /**
-   * Attempts to join this voice-based channel.
-   * @returns {Promise<VoiceConnection>}
-   * @example
-   * // Join a voice-based channel
-   * channel.join()
-   *   .then(connection => console.log('Connected!'))
-   *   .catch(console.error);
-   */
-  join() {
-    return this.client.voice.joinChannel(this);
-  }
-
-  /**
-   * Leaves this voice-based channel.
-   * @example
-   * // Leave a voice-based channel
-   * channel.leave();
-   */
-  leave() {
```

Comment:
Without join and leave, how are developers supposed to join/leave voice channels?

Reply:
That is done completely externally now, e.g. with @discordjs/voice